src/main/java/org/mule/service/http/impl/service/server/ErrorRequestHandler.java

```diff
@@ -37,7 +39,14 @@ public ErrorRequestHandler(int statusCode, String reasonPhrase, String entityFor
 
   @Override
   public void handleRequest(HttpRequestContext requestContext, HttpResponseReadyCallback responseCallback) {
-    String resolvedEntity = getResolvedEntity(requestContext.getRequest().getUri());
+    String resolvedEntity;
```

Comment:
This is only to indicate that the requested path does not exist or is unavailable, you don't need to expose anything more than that.